src/fetch/FetchExample.re

```diff
@@ -0,0 +1,93 @@
+type tDog = string;
+
+type tArrayDogs = array(tDog);
+
+type state = { dogs: tArrayDogs, isLoading: bool, error: bool };
+
+type action =
+  | DogFetch
```

Comment:
These should all be plural I think, since it's fetching mutliple dogs, not just a single dog.